source/features/quick-mention.tsx

```diff
@@ -31,6 +31,10 @@ function init(): void {
 	// The hovercard attribute avoids `highest-rated-comment`
 	const avatars = select.all(`.TimelineItem-avatar > [data-hovercard-type="user"]:first-child:not([href="/${getUsername()!}"], .rgh-quick-mention)`);
 	for (const avatar of avatars) {
+		if (select.exists('.minimized-comment', avatar.parentElement!.nextElementSibling!)) {
```

Comment:
Let's just query the common ancestor, I think:

```suggestion
		if (avatar.closest('.TimelineItem')!.querySelector('.minimized-comment')) {
```

👆 also I try to avoid `select()` when doing traversing because it has an unnatural reading order:


```
1.                       avatar
2.                              .parentElement
3.                                             .nextElementSibling
4. '.minimized-comment'